src/attributes.rst

```diff
@@ -1490,7 +1492,7 @@ Attribute ``proc_macro``
 
 :dp:`fls_u48dtmh97g`
 :t:`Attribute` :c:`proc_macro` shall apply to public :t:`[function]s` in the
-crate root.
+crate root of a :t:`proc-macro crate`.
```

Comment:
There is nothing wrong here, however the entry for "proc-macro crate" in the Glossary is mangled, it is missing a section heading.